packages/design-system/src/styles/components/_Dialog.scss

```diff
@@ -44,14 +44,18 @@ dialog.fixed {
   display: inline-block;
   margin-top: $spacer-6;
   max-width: var(--measure-base);
-  padding: var(--dialog__padding);
+  padding: 0;
   text-align: start;
   width: 95%; // provide space for the background layer to peek through
 
   @media (-ms-high-contrast: active), (forced-colors: active) {
     outline: $spacer-2 solid WindowText;
   }
 
+  > [role='document'] {
+    padding: var(--dialog__padding);
+  }
```

Comment:
Could you further explain the problem this is fixing? I don't see a problem with text inside the dialog being called out in the ticket, and I don't see it myself.

Reply:
This solves the problem of dialogs opening with their text at the top of the window. If you open the privacy settings dialog for example you get: 

<img width="754" alt="image" src="https://github.com/CMSgov/design-system/assets/783820/ea9a4813-70ef-4669-8ee4-122e725342a7">

With this fix it moves the container padding to the content making it appear 'correctly'.

Reply:
It might be something meant to solve our PrivacyModal VRT issue. 

I'm assuming Forrest discovered the `tabindex` on the `role="document"` meant it would sometimes scroll into focus. And since the padding was applied to the dialog container itself instead of this  element, the top padding was missing resulting in our VRTs showing differences. That's a guess though.

Reply:
Oh sweet! Nice!